src/hook/useAccumulator.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from 'react';
+
+const useAccumulator = () => {
+  const [accumulator, setAccumulator] = useState(0);
+  const accumulate = (amount) => {
+    setAccumulator(accumulator + amount);
+  };
+  return {
+    accumulator,
+    accumulate,
+  };
```

Comment:
```suggestion
  const [accumulator, setAccumulator] = useState(0);

  const accumulate = (amount) => {
    setAccumulator(accumulator + amount);
  };

  return {
    accumulator,
    accumulate,
  };
```

서로 관련이 있는 것들끼리 적절히 묶어주면 더 읽기 좋을 것 같아요